stdlib/experimental/prometheus/prometheus.flux

```diff
@@ -0,0 +1,3 @@
+package prometheus
+
+builtin from
```

Comment:
Minor nitpick. I think we should call this something else because we are not retrieving data from prometheus which is something we would likely want to support. This is acting as a prometheus metrics scraper so I think it needs a different name to signify that difference.

Reply:
Yeah I just changed this to scrape.